src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -62,6 +63,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('secret_build_arg', help="Secret build argument in 'name[=value]' format.", action='append', validator=validate_secret_build_arg)
         c.argument('no_push', help="Indicates whether the image built should be pushed to the registry.", arg_type=get_three_state_flag())
         c.argument('no_logs', help="Do not show logs after successfully queuing the build.", action='store_true')
+        c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', choices=SUPPORTED_OS_TYPES)
```

Comment:
You can move this existing definition here https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py#L126 and reuse it for build-task and build.